http/1.9/modules/ROOT/pages/http-connector-config-acb.adoc

```diff
@@ -0,0 +1,252 @@
+* *Realm*
++
+Authentication realm.
++
+*Security providers*
++
+This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:http-documentation.adoc#basicSecurityFilter[Basic Security Filter].
++
+* *Attributes*
++
+HTTP Request attributes that come from an HTTP Listener source and check the `Authorization` header. 
+
+
+=== Configuring Attributes for the HTTP Request
+
+The Anypoint Connector for HTTP (HTTP Connector) *Request* operation enables you to consume an external HTTP service using the default `GET` method. Otherwise, you can choose what methods the operation accepts
+
+To configure the attributes for HTTP *Request* in the Anypoint Code Builder UI:
+
+. Click the *Request* component on the canvas.
++
+The configuration UI opens.
+. In the *General* tab, configure the following attributes:
+
+* *Method*
++
+Comma-separated list of allowed HTTP methods. To allow all methods, do not define this attribute.
++
+* *URI Settings*
+** *URL*
++
+URL to send the request to.
++
+** *Path*
++
+Path for where to send the request.
++
+* *Request*
+** *Body*
++
+Body of the request response message.
++
+** *Headers*
++
+Headers to include in the response message.
++
+** *URI Parameters*
++
+URI parameters used to create the request.
++
+** *Query Parameters*
++
+Query parameters to include in the request.
++
+* *Correlations ID*
++
+Select an option for whether to include an outbound correlation ID in the request. 
++
+. Select the *Advanced* tab to configure the following attributes:
++
+* *Output MIME type*
++
+Defines the format of the request.
++
+* *Parameters*
++
+Add parameters to the request. 
++
+* *Output encoding*
++
+Select the encoding to transform the data to.
++
+* *Streaming Strategy*
++
+This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:connectors::streaming-strategies.adoc[]
++
+* *Connector Overrides*
+** *Follow redirects*
++
+Specifies whether or not to follow redirects.
++
+** *Send body mode*
++
+Select an option to define whether the request contains a body. If you select *Auto*, the behavior is determined by the defined allowed HTTP methods (GET, HEAD, and OPTIONS does not send a body).
++
+** *Request streaming mode*
++
+Select an option to define whether the request is sent using streaming.
++ 
+** *Response timeout*
++
+Maximum time in milliseconds that the request element blocks the execution of the flow waiting for the HTTP response.
++
+* *Response*
+** *Response validator*
++
+Configures default error handling for the request. This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:http-documentation.adoc#SuccessStatusCodeValidator[Success Status Code Validator] and xref:http-documentation.adoc#FailureStatusCodeValidator[Failure Status Code Validator].
+* *Output*
+** *Target Variable*
++
+Name of the variable on which the operation's output is placed.
```

Comment:
Maybe the no-verb version is the better choice.

Reply:
Hopefully I caught them all.